app/assets/stylesheets/osem.css.scss

```diff
@@ -55,7 +55,9 @@ fieldset {
   margin: 20px 0 20px 0;
 }
 
-.bootstrap-switch { height: 1.7em }
+.bootstrap-switch {
```

Comment:
That's a line comment on the PR! You don't lose it when you squash your commits!